### ISSUE TITLE
Change CLA to easier contribution option 

### ISSUE THREAD

**Issue by jasonblais** [jasonblais/mattermost-community#25]

**Comment by hanzei** [comment]
The problem I see is privacy. Why do contributors need to provide a phone number and an e-mail address? Providing a Mailing Address should also be done optionally.

Ideally we could get around of requesting a Full Name. But this could be impossible due to legal reasons.

**Comment by jasonblais** [comment]
Yeah, I wonder what information other open source projects require for the CLA.

One other reason we ask for the mailing address and such are that we send swag (Mattermost gifts) for them as a thank you for the contribution. But this could be optional,

**Comment by hanzei** [comment]
E.g. Hugo use https://cla-assistant.io/ and just require a GH username.
